.github/workflows/cpplint_modified_files.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   cpplint_modified_files:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1  # v2 is broken for git diff
+      - uses: actions/checkout@v1 # v2 is broken for git diff
```

Comment:
Please revert.  Two spaces is clearer to the reader.

Reply:
This is per the clang-format specs. But if this is a one-off change, will change it

Reply:
The is .yml, not .cpp ;-)